helpers/createTextCompletion.ts

```diff
@@ -0,0 +1,57 @@
+import { IHttp, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
+import { IUser } from '@rocket.chat/apps-engine/definition/users';
+import { notifyMessage } from './notifyMessage';
+import { App } from '@rocket.chat/apps-engine/definition/App';
+
+export async function createTextCompletion(
+	app: App,
+	room: IRoom,
+	read: IRead,
+	user: IUser,
+	http: IHttp,
+	prompt: string,
+	threadId?: string
+): Promise<string> {
+	try {
+		const model = await app
+			.getAccessors()
+			.environmentReader.getSettings()
+			.getValueById('model');
```

Comment:
Actually, you can read the setting by:
```
await read
            .getEnvironmentReader()
            .getSettings()
            .getValueById(MODEL_SELECTION_ID);
```

Please feel free to create an enum or const for setting strings.